.vscode/launch.json

```diff
@@ -0,0 +1,16 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Remote Debug Kind Operator",
+            "type": "go",
+            "request": "attach",
+            "mode": "remote",
+            "host": "127.0.0.1",
+            "port": 40000,
+            "remotePath": "",
+            "cwd": "${workspaceFolder}/example/sample-operator",
+            "trace": "log"
+        }
+    ]
+}
```

Comment:
Do we need this file?

Reply:
Yes. This is good to have to be able to connect to the pod running in the kind cluster (using vscode) to perform debugging on the running pod.